### ISSUE TITLE
dynamic trees don't work

### ISSUE THREAD

**Issue by Envel-Nikita-Gutsenkov** [the-realest-stu/Rustic#303]
dynamic trees do not affect rustic trees such as apple, ironwood, and olive. this is sad, although their support is declared

**Comment by ferreusveritas** [comment]
This is not true.  It works well.  Are you expecting the existing trees to become dynamic?  Please send a screenshot to help clarify your problem.

**Comment by Envel-Nikita-Gutsenkov** [comment]
https://i.ibb.co/GCmLg6S/javaw-b-Rc9m-Sxgd-P-compressed.jpg

+ addon dynamic trees TFC
